packages/cfd/src/Containers/trade-modal.tsx

```diff
@@ -0,0 +1,409 @@
+        default:
+            return getPlatformDXTradeDownloadLink(type);
+    }
+};
+
+const getTitle = (market_type: string, is_eu_user: boolean) => {
+    if (is_eu_user) localize('MT5 CFDs');
+    return market_type;
+};
+
+const DxtradeDesktopDownload = ({ dxtrade_tokens, is_demo }: TDxtradeDesktopDownloadProps) => {
+    return (
+        <React.Fragment>
+            <a
+                className='cfd-trade-modal__dxtrade-button'
+                href={getDXTradeWebTerminalLink(
+                    is_demo ? 'demo' : 'real',
+                    dxtrade_tokens && dxtrade_tokens[is_demo ? 'demo' : 'real']
+                )}
+                target='_blank'
+                rel='noopener noreferrer'
+            >
+                <Icon className='cfd-trade-modal__dxtrade-button-icon' icon='IcBrandDxtradeWordmark' size={36} />
+                <div className='cfd-trade-modal__dxtrade-button-text'>
+                    <Text color='colored-background' size='xxs' weight='bold'>
+                        <Localize i18n_default_text='Web terminal' />
+                    </Text>
+                </div>
+            </a>
+        </React.Fragment>
+    );
+};
+
+const DerivEZDesktopDownload = () => {
+    return (
+        <React.Fragment>
+            <a className='cfd-trade-modal__dxtrade-button' href={''} target='_blank' rel='noopener noreferrer'>
+                <Icon className='cfd-trade-modal__dxtrade-button-icon' icon='IcBrandDerivEzWordmark' size={36} />
+                <div className='cfd-trade-modal__dxtrade-button-text'>
+                    <Text color='colored-background' size='xxs' weight='bold'>
+                        <Localize i18n_default_text='Web terminal' />
+                    </Text>
+                </div>
+            </a>
+        </React.Fragment>
+    );
+};
+
+const CTraderDesktopDownload = () => {
+    return (
+        <React.Fragment>
+            <a
+                className='cfd-trade-modal__dxtrade-button'
+                href={'https://ct.deriv.com/'}
+                target='_blank'
+                rel='noopener noreferrer'
+            >
+                <Icon className='cfd-trade-modal__dxtrade-button-icon' icon='IcBrandCtraderWordmark' size={36} />
+                <div className='cfd-trade-modal__dxtrade-button-text'>
+                    <Text color='colored-background' size='xxs' weight='bold'>
+                        <Localize i18n_default_text='Web terminal' />
+                    </Text>
+                </div>
+            </a>
+        </React.Fragment>
+    );
+};
+
+const TradeModal = ({
+    mt5_trade_account,
+    is_eu_user,
+    onPasswordManager,
+    toggleModal,
+    dxtrade_tokens,
+    is_demo,
+    platform,
+}: TradeModalProps) => {
+    const differentQRCodes = (acc_type: TPlatformType) => {
+        if (acc_type === 'dxtrade') {
+            return (
+                <>
+                    <QRCode
+                        value={mobileDownloadLink(acc_type, 'android')}
```

Comment:
How about the rest of the platforms? 
There are QR codes that supports multiple operating systems.

Reply:
We are supposed to show just one QR code in our modal and in this case we can just have one platform shown.
We need to discuss it with the PM and design team.